docs/configuration/v2.x/metrics/express-route-circuits.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Azure Express Route Circuits Declaration
+---
+
+## Azure Express Route Circuits
+
+![Availability Badge](https://img.shields.io/badge/Available%20Starting-v1.0-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
```

Comment:
This will only be part of v2.0
```suggestion
![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.0-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
```